.circleci/config.yml

```diff
@@ -350,6 +350,17 @@ jobs:
       - store_artifacts:
           path: /home/circleci/.npm/_logs/
           destination: npm-logs
+      - run:
+          name: 'Create Incident on PagerDuty'
+          when: on_fail
+          command: |
+            curl -X POST --header 'Content-Type: application/json' -d '{
+              "service_key": "${PAGERDUTY_KEY}",
+              "event_type": "trigger",
+              "description": "FAILURE -- React Widgets Tap Test -- ${CIRCLE_BUILD_URL}",
+              "client": "CircleCI - React Widgets",
+              "client_url": "${CIRCLE_BUILD_URL}"
+            }' https://events.pagerduty.com/generic/2010-04-15/create_event.json
```

Comment:
Is there a reason why you didn't use Pager Duty [Event API v2](https://v2.developer.pagerduty.com/docs/send-an-event-events-api-v2)?

Reply:
v2 requires a "from" property which is essentially an email, this just uses the key as authentication which i think its just less setup and cleaner

Reply:
am I missing something? Where is the `from` parameter?
<img width="824" alt="Screen Shot 2020-03-27 at 10 12 30 AM" src="https://user-images.githubusercontent.com/6610308/77781965-b74a2280-7013-11ea-9d10-1d8e6e55cf04.png">